.travis.yml

```
@@ -23,9 +23,9 @@ env:
 stages:
   - test
   - name: website_dev
-    if: tag =~ ^v(\d+|\.)+[a-z]\d+$ OR tag = website_dev
+    if: (tag =~ ^v(\d+|\.)+[a-z]\d+$) OR (tag = website_dev) OR (commit_message =~ /^.*(website_dev).*$/)
```

```suggestion
    if: (tag =~ ^v(\d+|\.)+([a-z]|rc)\d+$) OR (tag = website_dev) OR (commit_message =~ /^.*(website_dev).*$/)
```

---

Or maybe:

```suggestion
    if: (tag =~ ^v(\d+|\.)+(a|b|rc)\d+$) OR (tag = website_dev) OR (commit_message =~ /^.*(website_dev).*$/)
```

---

Or, to make dev website builds a superset that includes release builds as well:

```suggestion
    if: (tag =~ ^v(\d+|\.)+(a|b|rc)?\d+$) OR (tag = website_dev) OR (commit_message =~ /^.*(website_dev).*$/)
```